wmt-etl/README.md

```diff
@@ -0,0 +1,39 @@
+# WMT Alpha Data Integration Prototype
+
+This section of the prototype illustrates how we will implement data integration.
+
+[Pandas](http://pandas.pydata.org/) is a flexible and powerful Python library for data manipulation and analysis.
+
+It provides rich functionality for working with database, file based and API abased data souces and for converting between them.
+
+### Set-up Python Dev environment
+Python development uses [virtualenvwrapper](http://virtualenvwrapper.readthedocs.io/en/latest/index.html) to provide dependency isolation.
+
+To set up Python and VirtualEnv install as follows:
+
+`brew install python`
+`pip install virtualenvwrapper`
+
+Add the following lines to your `~/.bash_profile`:
+
+```
+export WORKON_HOME=$HOME/.virtualenvs
+export PROJECT_HOME=$HOME/Devel
+source /usr/local/bin/virtualenvwrapper.sh
+```
+
+From the base project directory execute the following command to set up your environment:
+
+`mkvirtualenv -a wmt-etl -r wmt-spark/requirements.txt wmt-etl` 
```

Comment:
The requirements file path seems to be relative to the specified directory so `-r wmt-spark/requirements.txt` here should just be `-r requirements.txt`